test/unit/org/apache/cassandra/cql3/ViewComplexDeletionsTest.java

```diff
@@ -354,7 +356,7 @@ private void testCommutativeRowDeletion(boolean flush) throws Throwable
             ColumnFamilyStore cfs = ks.getColumnFamilyStore("mv");
             List<String> sstables = cfs.getLiveSSTables()
                                        .stream()
-                                       .sorted((s1, s2) -> s1.descriptor.generation - s2.descriptor.generation)
+                                       .sorted(Comparator.comparing(s -> s.descriptor.generation, SSTableUniqueIdentifierFactory.COMPARATOR))
```

Comment:
We can use `SSTableReader.generationComparator`:
```suggestion
                                       .sorted(SSTableReader.generationComparator)
```